docs/oscar_references.bib

```diff
@@ -175,6 +205,44 @@ @Book{Coh93
   url           = {https://doi.org/10.1007/978-3-662-02945-9}
 }
 
+@Misc{cohomCalg:Implementation,
+title={cohomCalg package},
+howpublished={Download link},
+url={http://wwwth.mppmu.mpg.de/members/blumenha/cohomcalg/},
```

Comment:
This redirects to https://github.com/BenjaminJurke/cohomCalg/releases ?

Reply:
Thx. Adjusted.